### DISCUSSION TITLE
ReSharper support

### DISCUSSION THREAD

**Discussion by sanghel-orbyta** [reqnroll/Reqnroll#215] [Ideas]
I've seen the [implemented support for Rider](https://github.com/reqnroll/Reqnroll.Rider?tab=readme-ov-file#reqnroll-support-for-resharper-and-rider).

I'm using ReSharper in VS2022, and would like support for reqnroll.

Particularly, at least support for "double-click test" > go to feature step, in the ReSharper UnitTestSessions window.
At the moment, double-clicking a test goes to the generated code-behind method for the step.

**Comment by gasparnagy** [comment]
Yes, this would be great.

I have also seen that the Rider plugin works also a R# plugin, but I don't know if this is really true. I have never tested myself.

Let's keep this idea open. Maybe someone have time to analyze that deeper.
